config.md

```diff
@@ -110,6 +110,26 @@ Note: Any OPTIONAL field MAY also be set to null, which is equivalent to being a
   The name of the operating system which the image is built to run on.
   Configurations SHOULD use, and implementations SHOULD understand, values listed in the Go Language document for [`GOOS`][go-environment].
 
+- **os.version** *string*, OPTIONAL
+
+  This OPTIONAL property specifies the version of the operating system targeted by the referenced blob.
+  Implementations MAY refuse to use manifests where `os.version` is not known to work with the host OS version.
+  Valid values are implementation-defined. e.g. `10.0.14393.1066` on `windows`.
+
+- **os.features** *array of strings*, OPTIONAL
+
+  This OPTIONAL property specifies an array of strings, each specifying a mandatory OS feature.
+  When `os` is `windows`, image indexes SHOULD use, and implementations SHOULD understand the following values:
+
+  - `win32k`: image requires `win32k.sys` on the host (Note: `win32k.sys` is missing on Nano Server)
+
+  When `os` is not `windows`, values are implementation-defined and SHOULD be submitted to this specification for standardization
```

Comment:
My only comment is that we don't have any other JSON fields that have `.` in their name, and this probably(?) will play silly buggers with `jq` commands, especially since there is a separate `os` field in the same struct. Can we call them `osVersion` and `osFeatures` or something similar (kinda sucks we can't make the `os` a struct without breaking compatibility).

Reply:
Also "SHOULD be submitted to this specification" is kind of a weird thing to put in a standard, since SHOULD is entirely about rules for implementations. I get this was copied from the original `variants` table text, but it would be a good idea to remove it.

Reply:
This was all copied from the `platform` definition in index.
Note that the fields in `platform` are `os.*` which is why I did this here.

I went ahead and updated this with your suggestion to camel case it.

Reply:
Ah I missed that. I just checked and you can get `jq` to deal with `.`s in field names so while I think `os.variant` might've been a mistake initially we should probably stick with it for consistency's sake -- sorry for the churn.

Reply:
No worries! Updated.